### ISSUE TITLE
ng-update: hammerjs v9 migration fails if project tsconfig does not specify explicit root file names

### ISSUE THREAD

**Issue by ahasall** [angular/components#18504]
#### Reproduction

ng update @angular/material

#### Expected Behavior

It should works without a warning or error

#### Actual Behavior

```
Using package manager: 'yarn'
Collecting installed dependencies...
Found 59 dependencies.
Fetching dependency metadata from registry...
    Updating package.json with dependency @angular/material @ "9.0.0" (was "8.2.3")...
    Updating package.json with dependency @angular/cdk @ "9.0.0" (was "8.2.3")...
UPDATE package.json (3974 bytes)
✔ Packages installed successfully.
** Executing migrations of package '@angular/material' **

❯ Updates Angular Material to v9
Could not find project to perform HammerJS v9 migration. Please ensure your workspace configuration defines a project.
✖ Migration failed. See above for further details.
```

#### Environment

- Angular: 9.0.1
- CDK/Material: 8.2.3

**Comment by devversion** [comment]
I don't think there is enough information for us to reproduce this issue. We didn't see this happen in various projects we tested the migration against. 

I see that we could improve the logic for finding the project for the migration, but that doesn't explain to why the error is thrown. Would love to get a reproduction for this. Maybe you could share the contents of your project `tsconfig` file?

**Comment by devversion** [comment]
Thx @ahasall for helping with reproducing this issue over VC. I've submitted a fix: #18525. See the PR for more details on what caused this error.

**Comment by ahasall** [comment]
You're welcome. Thanks for the fix.
